### PR TITLE
[DRAFT] Deprecations and Removals in Chrome 56

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/2016/12/chrome-56-deprecations.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+## Mouse on Android stops firing TouchEvents
+
+Until version 55, Android low-level mouse events in Chrome primarily followed an event path designed for touch interactions. For example, mouse drag motion while a mouse button is pressed generates `MotionEvents` delivered through `View.onTouchEvent`. 
+
+However, since touch events cannot support hover, hovering mousemoves followed a separate path. The whole design had quite a few side-effects including mouse interactions firing `TouchEvents`, all moue buttons appearing as *left* mouse buttons, and `MouseEvents` being suppressed by `TouchEvents`.
+
+Starting with Chrome 56, a mouse on Android M Or later will:
+
+* No longer fire `TouchEvents`.
+* Fire a cosistent sequence of `MouseEvents` with appropriate buttons and other properties.
```
Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/chrome-56-deprecations.md]: consistent

File: src/content/en/updates/2016/12/chrome-56-deprecations.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+[Chromestatus Tracker](https://www.chromestatus.com/feature/6131337345892352) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=611981)
+
+## Disallow all fetches for scripts with invalid type/language attributes
+
+Currently, Chrome's preload scanner fetches items in `<scripts>` elements regardless of the value of the `type` or `language` attribute, though the script will not be executed when parsed. By deprecating the fetch, the preload scanner and the parser will have the same semantics, and we will not be initiating fetches for scripts we will not use. This is intended to save data for users who navigate to sites with a lot of custom script tags that are post-processed (like `type=”text/template”`, for example).
+
+The use case of using invalid scripts to ping servers is adequately covered by the [sendBeacon API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon).
+
+This change alligns Chrome with Safari, though Firebox still requests scripts regardless of type or language.
```
Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/chrome-56-deprecations.md]: aligns

Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/chrome-56-deprecations.md]: Firefox ;)

File: src/content/en/updates/2016/12/chrome-56-deprecations.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/eu57SCNltls/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5760718284521472) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=626321&desc=2)
+
+## Remove MediaStreamTrack.getSources()
+
+This method is no longer part of the spec and is not supported by any other major browser. It has been replaced by `MediaDevices.enumerateDevices()`, which Blink has supported without flags since version 47 and which is also supported by other browsers.
```
Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/chrome-56-deprecations.md]: Maybe link to https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/enumerateDevices

File: src/content/en/updates/2016/12/chrome-56-deprecations.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome to help you plan. #}
+
+# API Deprecations and Removals in Chrome 56 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 56,
+which is in beta as of December xx. This list is subject to change at any time.
```
Comment (PaulKinlan) [inline src/content/en/updates/2016/12/chrome-56-deprecations.md]: xx == date.

**Pull request merged**